### ISSUE TITLE
install 3.3 version, create prometheus CRD fail

### ISSUE THREAD

**Issue by zeasly** [kubesphere/kubesphere#5012]
**What's your question**
Install v3.3 has error. System is running fine, but the monitor don't work.
```
    {

           "_ansible_item_label": "/kubesphere/kubesphere/prometheus/prometheus-operator/prometheus-operator-0prometheusCustomResourceDefinition.yaml",

           "_ansible_no_log": false,

           "ansible_loop_var": "item",

           "attempts": 5,

           "changed": true,

           "cmd": "/usr/local/bin/kubectl apply -f /kubesphere/kubesphere/prometheus/prometheus-operator/prometheus-operator-0prometheusCustomResourceDefinition.yaml --force || /usr/local/bin/kubectl apply -f /kubesphere/kubesphere/prometheus/prometheus-operator/prometheus-operator-0prometheusCustomResourceDefinition.yaml --force-conflicts --server-side",

           "delta": "0:00:04.117655",

           "end": "2022-06-29 15:55:38.827395",

           "failed": true,

           "failed_when_result": true,

           "invocation": {

             "module_args": {

               "_raw_params": "/usr/local/bin/kubectl apply -f /kubesphere/kubesphere/prometheus/prometheus-operator/prometheus-operator-0prometheusCustomResourceDefinition.yaml --force || /usr/local/bin/kubectl apply -f /kubesphere/kubesphere/prometheus/prometheus-operator/prometheus-operator-0prometheusCustomResourceDefinition.yaml --force-conflicts --server-side",

               "_uses_shell": true,

               "argv": null,

               "chdir": null,

               "creates": null,

               "executable": null,

               "removes": null,

               "stdin": null,

               "stdin_add_newline": true,

               "strip_empty_ends": true,

               "warn": true

             }

           },

           "item": "/kubesphere/kubesphere/prometheus/prometheus-operator/prometheus-operator-0prometheusCustomResourceDefinition.yaml",

           "msg": "non-zero return code",

           "rc": 1,

           "start": "2022-06-29 15:55:34.709740",

           "stderr": "The CustomResourceDefinition \"prometheuses.monitoring.coreos.com\" is invalid: metadata.annotations: Too long: must have at most 262144 bytes\nThe CustomResourceDefinition \"prometheuses.monitoring.coreos.com\" is invalid: metadata.annotations: Too long: must have at most 262144 bytes",

           "stderr_lines": [

             "The CustomResourceDefinition \"prometheuses.monitoring.coreos.com\" is invalid: metadata.annotations: Too long: must have at most 262144 bytes",

             "The CustomResourceDefinition \"prometheuses.monitoring.coreos.com\" is invalid: metadata.annotations: Too long: must have at most 262144 bytes"

           ],

           "stdout": "",

           "stdout_lines": []

         }
```

**Environment: OS & Hardware Information**

centos 7, 4C 8G * 2
k8s version is 1.21

**Error logs or message (Attach logs or screenshot)**

**Installer Version**

> v3.3

**Comment by junotx** [comment]
@zeasly 
This is related to https://github.com/kubernetes/kubectl/issues/712 fixed by https://github.com/kubernetes/kubernetes/pull/102105. 
Please use the k8s version >= v1.20.7 and >= 1.21.1

**Comment by wenxin-01** [comment]
Issue fixed in `ks v3.3.1-rc.3`
